android/local.properties

```diff
@@ -1,5 +1,5 @@
-sdk.dir=/Users/drewnicolette/Library/Android/sdk
-flutter.sdk=/Users/drewnicolette/flutter
+sdk.dir=/Users/daryleurrea/Library/Android/sdk
```

Comment:
We should keep an eye on this in case it needs to be specified for each developers specific SDK path